tajo-docs/src/main/sphinx/table_management/text.rst

```diff
@@ -1,21 +1,21 @@
 *************************************
-CSV (TextFile)
+TEXT
 *************************************
 
-A character-separated values (CSV) file represents a tabular data set consisting of rows and columns.
+A character-separated values text file represents a tabular data set consisting of rows and columns.
```

Comment:
It would be better if readers can easily understand what text means. As you know, ```text``` can represent both the TEXT file and strings.
How about emphasize the letters ```text``` when it means the TEXT file? For sphyinx syntax, please refer to http://sphinx-doc.org/rest.html.